htdocs/install/mysql/migration/16.0.0-17.0.0.sql

```diff
@@ -65,6 +65,7 @@ ALTER TABLE llx_overwrite_trans ADD UNIQUE INDEX uk_overwrite_trans(lang, transk
 ALTER TABLE llx_mailing_cibles MODIFY COLUMN source_type varchar(32); 
 
 ALTER TABLE llx_actioncomm ADD INDEX idx_actioncomm_percent (percent);
+ALTER TABLE llx_paiementcharge ADD COLUMN entity integer NOT NULL DEFAULT 1 AFTER rowid;
```

Comment:
No database structure change is allowed into an already released version. We can return in the past. What was released is already released.
Also there is no reason to have entity field on a payment table. A payment is always in entity of its parent object.

Reply:
please check https://github.com/Dolibarr/dolibarr/pull/26034